.github/workflows/main.yml

```
@@ -0,0 +1,23 @@
+name: CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java-version:
+          - 8
+          - 11
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java-version }}
+      - name: Maven install dependencies
+        run: mvn install -DskipTests -Dair.check.skip-all
+      - name: Maven run tests
+        run: mvn test
```

```suggestion
        run: mvn -B install -DskipTests
      - name: Maven run tests
        run: mvn -B test
```

Let's use Maven's batch mode to ensure output is more suitable for CI (no download progress etc.)

Also why pass `-Dair.check.skip-all`? Was it because checkstyle failed locally for you?

---

> Let's use Maven's batch mode to ensure output is more suitable for CI (no download progress etc.)
done
> Also why pass -Dair.check.skip-all? Was it because checkstyle failed locally for you?
Yeah, that was the reason. Removed.